packages/synapse-react-client/src/index.ts

```diff
@@ -1,24 +1,126 @@
-import { displayToast } from './components/ToastMessage'
+import * as SynapseComponents from './components'
```

Comment:
The dts generator doesn't allow us to use `export * from './file'` at the top level, so I had to go through and manually import/export each identifier that we rely on today.

Identifying how we should structure and maintain this is an exercise for the future, since doing anything else could result in breaking changes that I don't want to risk in this PR.